scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -683,12 +683,29 @@ public PrivateKey getPrivateKey() {
 	 * Gets the registry of <em>shared secrets</em> used for authenticating
 	 * clients during a DTLS handshake.
 	 * 
-	 * @return the registry
+	 * @return the registry. Maybe {@code null}, if a advanced psk store is
+	 *         provided to the builder.
+	 * @deprecated use {@link #getAdvancedPskStore()} instead
 	 */
+	@Deprecated
 	public PskStore getPskStore() {
 		return pskStore;
 	}
 
+	/**
+	 * Gets the advanced registry of <em>shared secrets</em> used for
+	 * authenticating clients during a DTLS handshake.
+	 * 
+	 * If a deprecated {@link PskStore} is provided to the builder, a
```

Comment:
getPskStore itself is not longer used. I f someone calls that on a incomplete configuration in order to check, if a psk-store is set, that will fail, if the advanced psk store is used directly. Until now, only the psk sotre is used, so for now, that should not cause problems with already existing implementations.

About deprecating the PskStore itself, I'm not sure, because the new store may just use it for the "synchronous part of the psk store".

Reply:
I'm ok about deprecrated the getPskStore().

I'm ok too about we can keep the interface(pskStore) and setter(config.setPskStore) but in this case we should removed the "deprecated" word in this comment.
(or we keep the comment and  we deprecate the interface and the setter)